dspace-xmlui/src/main/java/org/dspace/app/xmlui/aspect/versioning/VersionManager.java

```diff
@@ -83,6 +83,7 @@ public static FlowResult processCreateNewVersion(Context context, UUID itemID, S
             context.abort();
             throw new RuntimeException(ex);
         }
+        context.complete();
```

Comment:
Looking more closely at this code, I don't understand why this `context.complete();` is needed in `processCreateNewVersion()` while it doesn't exist any of the other methods in `VersionManager` (and all the methods seem rather similar in nature).

Looking more closely at the code around this, `processCreateNewVersion()` is called by `doCreateNewVersion()` here: https://github.com/DSpace/DSpace/blob/dspace-6_x/dspace-xmlui/src/main/resources/aspects/Versioning/versioning.js#L127

And `doCreateNewVersion()` is called here: https://github.com/DSpace/DSpace/blob/dspace-6_x/dspace-xmlui/src/main/resources/aspects/Versioning/versioning.js#L101 , followed by a `getDSContext().complete();` just a few lines down.

So, I'd like to better understand why the existing code isn't working...it *seems* to me like we already should be calling `complete()` on the Context object.  Are there specific scenarios where this isn't happening?  Are we creating a new Context somewhere along the way that isn't getting committed?